.github/workflows/haskell.yml

```diff
@@ -1,143 +1,82 @@
 name: Haskell CI
 
-on: [push]
+on:
+  push:
+  pull_request:
+  schedule:
+    # Everyday at 4:00 AM
```

Comment:
Do we know the TZ? is this UTC?

Reply:
Great question! It appears to be [UTC](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onschedule). Is there a better time?

Reply:
Let's just document that it's 4am utc :-) 4 am is such an ambiguous value without a TZ.

Reply:
Done